examples/example5.jl

```diff
@@ -101,43 +101,46 @@ add!(m, DemandFunction(GOVT, 0.,
  [Demand(PA , 35.583)],
  [Endowment(PA, :($g0*$TAU_LS)), Endowment(PA, dtax), Endowment(PFX, bopdef)]))
 
```

Comment:
This is just reordering to put HH before GOVT, and some commenting out HH values for testing.